### PR TITLE
Update CHANGELOG.rst

### PR DESCRIPTION
**Pull request by t20100**
This PR updates the changelog with all pull requests since v0.11.0.

Please improve/modify.

### CODE REVIEW

**Comment by vallsv** [comment]
Woow, big bunch of things...

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,72 @@
...
+    * Added `silx.gui.widgets.ColormapNameComboBox` widget (PR #2814)
+    * Added `silx.gui.widgets.LegendIconWidget` widget (PR #2783)
+    * Added `silx.gui.utils.blockSignals` context manager (PR #2697, #2702)
+    * Added `silx.gui.utils.qtutils.getQEventName` function (PR #2725)
+    * Added `silx.gui.colors.asQColor` function (PR #2753)
+    * Minor fixes (PR #2662, #2667, #2674, #2719, #2724, #2747, #2757, #2760, #2766, #2789, #2798, #2799, #2805, #2811)
+
+* `silx.opencl`:
+
+  * Improved `silx.opencl.sparse.CSR` to support more data types (PR #2671)
```
Comment (pierrepaleo) [inline CHANGELOG.rst]: The sparse.CSR feature was not announced in `0.11.0`, perhaps it is time now to advertise it ?

Comment (t20100) [reply CHANGELOG.rst]: If it's ready yes!

**Comment by t20100** [comment]
That's quite a bit of a change: last release was 6 months ago (instead of 3-4 before), and I put a reference to all PRs: what do you thing about it?
The previous changelogs were a summary of the main changes.

**Pull request merged**